lib/govuk_app_config/govuk_content_security_policy.rb

```diff
@@ -75,8 +75,8 @@ def self.build_policy(policy)
                        "www.tax.service.gov.uk",
                        # Allow JSON call to Nuance - HMRC web chat provider
                        "hmrc-uk.digital.nuance.com",
-                       # Allow JSON call to klick2contact - HMPO web chat provider
-                       "hmpowebchat.klick2contact.com",
+                       # Allow JSON call to Eckoh - HMPO web chat provider
+                       "omni.eckoh.uk",
```

Comment:
Hmmm. Perhaps you should keep both entries until the new provider goes live?

Reply:
@leenagupte The new provider is live already, due to communication slowdown we're actually behind HMPO on this rollout.

Reply:
Sorry, I meant in our code.